source/shared/cpp/AdaptiveCardsSharedModel/AdaptiveCardsSharedModelUnitTest/AdaptiveCardsSharedModelUnitTest.cpp

```diff
@@ -0,0 +1,209 @@
+#include "stdafx.h"
```

Comment:
Would it make sense to name this file DateAndTimeUnitTests.cpp instead?

Reply:
thanks

---
In reply to: [150952950](https://github.com/Microsoft/AdaptiveCards/pull/871#discussion_r150952950) [](ancestors = 150952950)